quickwit/quickwit-core/src/lib.rs

```diff
@@ -68,7 +68,7 @@ mod tests {
         }
         // Now delete the index.
         let index_service =
-            IndexService::new(test_sandbox.metastore(), StorageUriResolver::for_test());
+            IndexService::new(test_sandbox.metastore(), StorageResolver::unconfigured());
```

Comment:
is `StorageResolver::unconfigured()` really unconfigured? Sound like `StorageUriResolver::for_test()` is doing more and that's what I would expect for a test config.

Reply:
They're equivalent. They both register the same factories with no config for the former and the default config for the latter. No config is also equivalent to the default config after this change.